ledger/ledger.go

```diff
@@ -862,6 +862,14 @@ func (l *Ledger) Validate(ctx context.Context, blk bookkeeping.Block, executionP
 	return &vb, nil
 }
 
+// DBRound returns the trackers' dbRound which "is always exactly accountsRound()"
+func (l *Ledger) DBRound() basics.Round {
```

Comment:
exposing db Round does not look a right idea but anyway, consider a name like `LatestTrackerCommitted` or similar to by on pair with existing `Latest()` and `LatestCommitted()`

Reply:
c43f6460d8e1f8b37796f0644e35ae470331c7c0